index.bs

```diff
@@ -1441,16 +1453,17 @@ The following operations can be invoked by the client in an authenticator sessio
 This operation must be invoked in an authenticator session which has no other operations in progress. It takes the following
 input parameters:
 
-- The caller's RP ID, as determined by the user agent and the client.
+- The caller's [=RP ID=], as optionally decared by the [=[RP]=] and vetted by the user agent and the client, or defaulted to the
```

Comment:
I think I'd prefer the original wording with "determined" linked to the step that determines it (you can put `id`s on `<li>` tags). The authenticator operations don't need to care much how the RP ID was determined, so we don't need to clutter their interfaces by re-describing it.